### DISCUSSION TITLE
Prevent assets being embedded into HTML files as base64 string?

### DISCUSSION THREAD

**Discussion by mwskwong** [gatsbyjs/gatsby#35520] [Help]
Self-explanatory title.

I think it is related to Webpack but I'm not sure which Webpack setting causes this to happen.

**Comment by mwskwong** [comment]
Bump

**Comment by LekoArts** [comment]
You can use the static folder (https://www.gatsbyjs.com/docs/how-to/images-and-media/static-folder) or use the `raw-loader`
